workspace.go

```diff
@@ -846,6 +855,35 @@ func (s *workspaces) SafeDeleteByID(ctx context.Context, workspaceID string) err
 	return req.Do(ctx, nil)
 }
 
+// RemoveAutoDestroyAt removes auto destroy from a workspace.
+func (s *workspaces) RemoveAutoDestroyAt(ctx context.Context, organization, workspace string) (*Workspace, error) {
+	if !validStringID(&organization) {
+		return nil, ErrInvalidOrg
+	}
+	if !validStringID(&workspace) {
+		return nil, ErrInvalidWorkspaceValue
+	}
+
+	u := fmt.Sprintf(
+		"organizations/%s/workspaces/%s",
+		url.QueryEscape(organization),
+		url.QueryEscape(workspace),
+	)
+
+	req, err := s.client.NewRequest("PATCH", u, &workspaceRemoveAutoDestroyAtOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	w := &Workspace{}
+	err = req.Do(ctx, w)
+	if err != nil {
+		return nil, err
+	}
+
+	return w, nil
```

Comment:
I personally like this approach much more, given the significance of a `null` value for this field.